js/utilities.js

```diff
@@ -9,15 +9,15 @@ export const showToastMessage = (message) => {
 };
 
 export const handleInputChange = ($inputField, $updateButton, currentTask) => {
-    const trimmedValue = $inputField.value.trim();
-    $updateButton.disabled = !(
-      trimmedValue.length > 0 && trimmedValue !== currentTask.title
-    );
-  };
+  const trimmedValue = $inputField.value.trim();
+  $updateButton.disabled = !(
+    trimmedValue.length > 0 && trimmedValue !== currentTask.title
+  );
+};
```

Comment:
Please follow the task description of the Trello board.

Reply:
I didn't get, what you intended to make me understand, bhai !

Reply:
solved